src/myspace/AdditionalInfo.jsx

```diff
@@ -1,21 +1,143 @@
-import React from "react";
+import React, { useState, useEffect, useContext } from "react";
+import { BASE_API } from "../config";
+import { AuthContext } from "../AuthContext";
+import "./MySpace.css";
 
 export default function AdditionalInfo() {
-    return (
-        <div className="container-fluid" id="profile">
-          <div className="top">
-            <h1>
-              This is the Private page for Member's Additional Info page
-            </h1>
-          </div>
-          <div className="middle">
-            <h2>The site is currently under construction...</h2>
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [additionalInfo, setAdditionalInfo] = useState({});
+  const { access_token, user } = useContext(AuthContext);
```

Comment:
We might want to persist the access_token as accessToken on our side. Because the backend might me using snake case, but on the frontend it's camel case everywhere. That would be a change in the context, I don't mind if you want to do that change separately. So I won't make that a blocker for this PR.
Small things, but consistent casing for the code hygiene is important

Reply:
Ok, can some else pick it up (like first-timer-only)? or do you want me to work on it?

Reply:
We can create a separate one and label as first timer I think :)